ipcontroller_config.py

```diff
@@ -238,7 +238,7 @@
 
 # The IP address for registration.  This is generally either '127.0.0.1' for
 # loopback only or '*' for all interfaces.
-# c.HubFactory.ip = u''
```

Comment:
Then it's probably the 'location' getting the wrong value.

When you give the ip '*', IPython picks one of the IP addresses of the machine to identify it (called 'location'). When other systems try to connect, they use this IP. It's likely that it's picking up the wrong value from a login node that has several network interfaces.  `c.IPControllerApp.location` allows you to set this explicitly.